DeAceroDLabs/marketplace-frontend
# [DMP-566] Add button to open Ventana Banorte

## What is done

[[DMP-566}] Importa scripts para abrir la ventana de pagos de Banorte](https://deacero.atlassian.net/browse/DMP-566)

## How is done

1. Created a new hook to use scripts in components.
2. Created a new component that implements the Button component.
3. Use the new hook to load the scripts for the integration with Banorte.
4. Starts the process to open the payment window.

**Note: This PR still has many errors, these are related to Banorte Documentation. They will be solved in the following tickets (I hope so)**

## How to test

1. Run the project via `npm start`
2. Go to the `carrito/pago` page.
3. Click on the "Realiza tu pedido y paga" button.
4. You should view the Banorte Popup displaying an error. (Probably you will need to click the button multiple times).
